ExtraDry/ExtraDry.Core/Slug.cs

```diff
@@ -181,6 +181,17 @@ public static string RandomWebString(int count)
         return RandomCharacters(count, characters);
     }
 
+    /// <summary>
+    /// Used to determine if a character can be used in a slug.
+    /// </summary>
+    /// <remarks>
+    /// Used instead of char.IsLetterOrDigit becuase that doesn't discriminate on charater set, which can allow invalid characters for urls
+    /// </remarks>
+    private static bool IsLatinLetterOrDigit(char c)
```

Comment:
It would be more accurate to call this IsAsciiLetterOrDigit which is the URL specification, even though that is equivelent to the Unicode Basic Latin character set.

Reply:
I've renamed from Latin to Ascii